recipes/preact-cli/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.5.1" %}
+
+package:
+  name: preact-cli
+  version: {{ version }}
+
+source:
+  url: https://registry.npmjs.org/preact-cli/-/preact-cli-{{ version }}.tgz
+  sha256: 029392799c88e65b2004c6569c7046fafa6547c83e4df265ad8a4675d95ba173
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
On the feedstock, you should be able to have a `skip: true  # [nodejs != '20']` and can then remove all of the pins below.

Reply:
This worked here as well!  I've also applied this to the other PR